Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -353,7 +356,7 @@ public static bool HasContents(MicrowaveComponent component)
         /// It does not make a "wzhzhzh" sound, it makes a "mmmmmmmm" sound!
         /// -emo
         /// </remarks>
-        public void Wzhzhzh(EntityUid uid, MicrowaveComponent component, EntityUid? user)
+        public void StartCooking(EntityUid uid, MicrowaveComponent component, EntityUid? user)
```

Comment:
?

Reply:
wzhzhzh perfectly describes the spaghetti that drives the microwave though

Reply:
pretty much

Reply:
> ?

If the change isn't warranted then I can revert it, lmk.

Reply:
you should revert it. it's hard to explain, but this function header must rename as it was.

Reply:
unacceptable breaking change to microwave api